static/templates/left_sidebar_stream_setting_popover.hbs

```diff
@@ -0,0 +1,12 @@
+<ul class="nav nav-list">
+    <li>
+        <a href="#streams/subscribed">
+            {{t "Browse streams" }}
+        </a>
+    </li>
+    <li>
+        <a href="#streams/new">
+            {{t "Create a stream" }}
+        </a>
+    </li>
```

Comment:
We should probably check `can_create_streams` to decide whether this option is displayed.